### PR TITLE
Refactor/denoising

### PR DESCRIPTION
**Pull request by simedroniraluca**

### CODE REVIEW

**Comment by simedroniraluca** [comment]
https://github.com/orgs/OceanStreamIO/projects/6/views/1?pane=issue&itemId=40962065

File: oceanstream/L2_calibrated_data/noise_masks.py [hunk 1]
```diff
@@ -172,45 +171,43 @@ def attach_mask_to_dataset(Sv: xarray.Dataset, mask: xarray.Dataset):
...
     for mask in masks:
         Sv = attach_mask_to_dataset(Sv, mask)
     return Sv
 
 
+def dict_to_formatted_list(d: Dict[str, Union[int, str]]) -> List[str]:
+    """Convert a dictionary to a list of formatted strings."""
+    return [f"{key}={value}" for key, value in d.items()]
+
+
```
Comment (ruxandra-valcu) [inline oceanstream/L2_calibrated_data/noise_masks.py]: Maybe add this to an utils file somewhere?

File: oceanstream/L2_calibrated_data/noise_masks.py [hunk 2]
```diff
@@ -236,17 +236,20 @@ def create_noise_masks_rapidkrill(source_Sv: xarray.Dataset):
...
         "dilate": [(5, 5), (7, 7)],
         "median": [(7, 7)],
     }
-
     impulse_mask = create_impulse_mask(source_Sv, parameters=impulse_mask_param, method="wang")
-    impulse_mask = add_metadata_to_mask(mask=impulse_mask, metadata={"mask_type": "impulse"})
+    impulse_mask = add_metadata_to_mask(
+        mask=impulse_mask, metadata={"mask_type": "impulse", "method": "wang"}
+    )
+
```
Comment (ruxandra-valcu) [inline oceanstream/L2_calibrated_data/noise_masks.py]: We're removing wang, so switch this to ryan

File: tests/test_noise_mask.py [hunk 3]
```diff
@@ -1,84 +1,105 @@
...
 
-def test_impulse(sv_dataset_jr230):
-    source_Sv = sv_dataset_jr230
+def test_impulse(ed_ek_60_for_Sv):
+    source_Sv = compute_sv(ed_ek_60_for_Sv)
     RYAN_DEFAULT_PARAMS = {"thr": 10, "m": 5, "n": 1}
     mask = create_impulse_mask(source_Sv, parameters=RYAN_DEFAULT_PARAMS)
     assert mask["channel"].shape == (3,)
 
 @pytest.mark.ignore
```
Comment (mihaiboldeanu) [inline tests/test_noise_mask.py:23]: This is one of the ignored tests by gitactions.

File: tests/test_noise_mask.py [hunk 4]
```diff
@@ -1,84 +1,105 @@
...
-    source_Sv = complete_dataset_jr179
+    source_Sv = compute_sv(ed_ek_60_for_Sv)
     mask = create_seabed_mask(source_Sv)
     add_metadata_to_mask(mask, metadata)
     Sv_mask = attach_mask_to_dataset(source_Sv, mask)
     for k, v in metadata.items():
         assert Sv_mask["mask_seabed"].attrs[k] == v
     assert Sv_mask["mask_seabed"].attrs["mask_type"]
 
-@pytest.mark.ignore
```
Comment (mihaiboldeanu) [inline tests/test_noise_mask.py:80]: This is one of the ignored tests by gitactions.

**Review by mihaiboldeanu — APPROVED** [review]
Looks good!

**Review by mihaiboldeanu — APPROVED** [review]

**Pull request merged**